presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/thrift/InMemoryThriftMetastore.java

```diff
@@ -283,6 +283,23 @@ public synchronized void alterTable(String databaseName, String tableName, Table
         return tables.build();
     }
 
+    @Override
+    public synchronized List<String> getTablesWithParameter(String databaseName, String parameterKey, String parameterValue)
+    {
+        requireNonNull(parameterKey, "parameterKey is null");
+        requireNonNull(parameterValue, "parameterValue is null");
+
+        ImmutableList.Builder<String> tables = ImmutableList.builder();
+        for (Map.Entry<SchemaTableName, Table> entry : relations.entrySet()) {
```

Comment:
Nit: this can be simplified to
```java
relations.forEach((tableName, table) -> {

Reply:
Thanks for the pointer, but I used `relations.entrySet().stream()` instead so that a `ImmutableList.Builder` is not needed anymore.

Reply:
Nice, even better